includes/data-stores/class-wcs-subscription-data-store-cpt.php

```diff
@@ -598,6 +598,6 @@ public function delete_all_metadata_by_key( $meta_key ) {
 	 * @return array
 	 */
 	public function get_subscriptions_count_by_status() {
-		return (array) wp_count_posts( 'shop_subscription' );
+		return WC_Data_Store::load( 'subscription' )->get_subscriptions_count_by_status();
```

Comment:
In testing, it looks like this function is calling itself recursively, resulting in the page endlessly loading.
This occurs when HPOS is disabled (using the WordPress posts table) and rendering the **WC → Status** page.

Reply:
Biggest facepalm 🤦‍♂️ This definitely shouldn't have been updated

Reply:
Fixed in [7d87fae](https://github.com/Automattic/woocommerce-subscriptions-core/pull/388/commits/7d87fae5f0d8eb95447829202b9e1c69a53c6665)